google/bigtable/bigtable.yaml

```diff
@@ -0,0 +1,34 @@
+# Google Bigtable API service configuration
+
+type: google.api.Service
+config_version: 0
```

Comment:
These are derived from the Google-internal source of truth; I think they have effects in API compiler but am not positive.